apps/admin/backend/schema.sql

```diff
@@ -1,44 +1,74 @@
 create table elections (
-  id serial primary key,
+  id text primary key,
   data text not null,
-  is_official_results boolean not null default false,
-  created_at timestamp not null default current_timestamp,
-  deleted_at timestamp
-);
+  is_official_results integer not null default false,
```

Comment:
Worth adding a check here?
```suggestion
  is_official_results integer not null default false
    check (is_official_results = true or is_official_results = false),
```